src/main/java/org/umssdiplo/automationv01/core/customwebdriver/Chrome.java

```diff
@@ -2,10 +2,16 @@
 
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
 
 public class Chrome implements CustomWebDriver {
     @Override
     public WebDriver generateWebDriver() {
-        return new ChromeDriver();
+        System.setProperty("webdriver.chrome.driver","/usr/bin/chromedriver");
```

Comment:
you can keep those changes in your STASH.

avoid to upload this implementation because your team will be affect with this change.